frontend/src/scenes/organization/Settings/Invites.tsx

```diff
@@ -46,72 +49,83 @@ function makeActionsComponent(
-                return <ProfilePicture name={invite.first_name} email={invite.target_email} />
-            },
-            width: 32,
-        },
-        {
-            title: 'Target Email',
-            dataIndex: 'target_email',
-            key: 'target_email',
-            render: function TargetEmail(target_email: string | null): JSX.Element | string {
-                return target_email ?? <i>none</i>
+            render: function TargetEmail(_, invite): JSX.Element | string {
+                return invite.target_email ? (
+                    <div className="flex-center">
+                        <ProfilePicture
+                            name={invite.first_name}
+                            email={invite.target_email}
+                            size="md"
+                            style={{ marginRight: 4 }}
+                        />
+                        {invite.target_email}
+                    </div>
+                ) : (
+                    <i>no one</i>
+                )
             },
         },
         {
             title: 'Created At',
             dataIndex: 'created_at',
             key: 'created_at',
-            render: (created_at: string) => humanFriendlyDetailedTime(created_at),
-        },
-        {
-            title: 'Created By',
-            dataIndex: 'created_by',
-            key: 'created_by',
-            render: (createdBy?: UserBasicType) => (createdBy ? `${createdBy.first_name} (${createdBy.email})` : '–'),
+            render: (_, { created_at }) => humanFriendlyDetailedTime(created_at),
         },
+        createdByColumn() as LemonTableColumn<OrganizationInviteType, keyof OrganizationInviteType | undefined>,
         {
             title: 'Invite Link',
             dataIndex: 'id',
             key: 'link',
-            render: (id, invite) => InviteLinkComponent(id as string, invite),
+            render: (_, invite) => InviteLinkComponent(invite.id, invite),
         },
         {
             title: '',
-            dataIndex: 'actions',
             key: 'actions',
             render: makeActionsComponent(deleteInvite),
         },
     ]
 
     return (
         <div>
-            <h2 className="subtitle" style={{ justifyContent: 'space-between' }}>
-                Pending Invites
-                {!!invites.length && <CreateInviteModalWithButton />}
+            <h2 id="invites" className="subtitle" style={{ justifyContent: 'space-between' }}>
+                Team invites
+                <Button type="primary" icon={<PlusOutlined />} onClick={() => setInvitingModal(true)}>
+                    Invite team member
+                </Button>
             </h2>
-            <Table
+            {!preflight?.email_service_available && (
+                <InfoMessage style={{ marginTop: 16 }}>
+                    <div>
+                        Sending emails is not enabled in your PostHog instance. Remember to{' '}
```

Comment:
Could this get a link to setup instructions somewhere?

Reply:
Added.